.github/workflows/pw_aws_ci.yaml

```
@@ -72,9 +72,9 @@ jobs:
       id: get-branch
       run: |
         pr_number=${{ github.event.inputs.pr_number }}
+        repo={{ github.repository }}
         if [ "$pr_number" -eq "0" ]; then
           branch=${{ github.event.inputs.ref }}
-          repo=${{ github.repository }}
```

I'm a little confused by the syntax used here with `{{}}` and `$`.  Is this Jinja or is this specific syntax for GitHub workflows?

To retain consistency, I'm guessing that this should be

```suggestion
        repo=${{ github.repository }}
        if [ "$pr_number" -eq "0" ]; then
          branch=${{ github.event.inputs.ref }}
```

---

Yes ${{ gitsutff }} is specific GitHub syntax, and yes line 75 was a typo